src/PrestaShopBundle/Resources/views/Admin/Common/Grid/grid_panel.html.twig

```diff
@@ -22,12 +22,16 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
+{% set displayName = true %}
+{% if (grid.view_options.display_name is defined) %}
+    {% set displayName = grid.view_options.display_name %}
+{% endif %}
 
 <div class="card js-grid-panel" id="{{ grid.id }}_grid_panel">
   {% block grid_panel_header %}
-    {% if grid.name is not null or grid.actions.grid|length > 0 %}
+    {% if displayName is not sameas(false) or grid.actions.grid|length > 0 %}
     <div class="card-header js-grid-header">
-      {% if grid.name is not null %}
+      {% if displayName is not sameas(false) %}
```

Comment:
```suggestion
      {% if displayName is sameas(true) %}
```

Reply:
I go with the principle that displayName is undefined or set to false
If I compare with true, I must set a default value to true

Reply:
@sowbiba so, your condition will be true if `displayName` is `true`, `undefined` or whatever value if strictly different to `false`, right?

Reply:
exactly